releasenotes/notes/moved-system-constants-7894698d5734235a.yaml

```diff
@@ -0,0 +1,9 @@
+---
+upgrade:
+  - |
+     Instead of using system constants from ``ddtrace.ext.system``. Use constants from ``ddtrace.constants`` module. 
+     For Example:: ``ddtrace.ext.system.PID`` -> ``ddtrace.constants.PID``
```

Comment:
```suggestion
     For example: ``ddtrace.ext.system.PID`` -> ``ddtrace.constants.PID``
```

Reply:
Also, there's only the one constant so I don't think we need to say "for example".

I think we can trim it down to

>  ``ddtrace.ext.system`` has been deprecated. Use ``ddtrace.constants.PID`` instead of ``ddtrace.ext.system.PID``, which will be removed in v1.0.